docs/performance/inserts/tuning.rst

```diff
@@ -100,6 +100,20 @@ gets flushed after every operation. Setting this to ``ASYNC`` will improve
 insert performance, but it also worsens durability. If a node crashes before a
 translog has been synced, those opperations will be lost.
 
+Overload Protection
+-------------------
+
+The `Overload Protection`_ settings control how many resources operations like 
+``INSERT INTO FROM QUERY`` or ``COPY`` can use.
+
+The default values serve as a starting point for an algorithm that dynamically 
+adapts the effective concurrency limit based on the round-trip time of requests. 
+Whenever one of these settings is updated, the previously calculated effective 
+concurrency is reset.
+
+Please update the settings accordingly, especially if you are benchmarking insert
+performance.
+
```

Comment:
A few trailing spaces sneaked in here. Can I humbly ask you to configure your editor to strip those, or take extra care when committing?

![image](https://github.com/crate/crate-howtos/assets/453543/d6d131ea-91ee-4050-b500-19906ae99ebe)